source-code/ide-extension/src/commands/openInEditor.ts

```diff
@@ -0,0 +1,17 @@
+import { getGitOrigin } from '../services/telemetry/implementation.js'
+import * as vscode from "vscode"
+
+const EDITOR_BASE_PATH = 'https://inlang.com/editor/'
+
+export const openInEditorCommand = {
+  id: "inlang.openInEditor",
+  title: "Inlang: Open in editor",
+  callback: async function ({ messageId }: { messageId?: string }) {
+    // TODO: Probably the origin should be configurable via the config.
```

Comment:
@openscript Nice, you included a comment for this to be configurable. Let's see how it envolves.